packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -78,7 +78,8 @@ class VMService {
         final bool pause = params.asMap['pause'] ?? false;
 
         if (isolateId is! String || isolateId.isEmpty)
-          throw new rpc.RpcException.invalidParams('Invalid \'isolateId\': $isolateId');
```

Comment:
This file seems to only contain formatting changes. Can we revert this file?

Reply:
Yes.  Not sure how that happened :confused:

Reply:
Done.